src/jest/JestFileReporter.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable class-methods-use-this */
+const VerboseReporter = require('@jest/reporters/build/verbose_reporter').default;
+const stripAnsi = require('strip-ansi');
+const fs = require('fs');
+const path = require('path');
+
+const filePath = path.resolve(__dirname, '../../tests/jest/reports/coverage/JestTestResults.txt');
+
+class JestFileReporter extends VerboseReporter {
+  // eslint-disable-next-line no-useless-constructor
+  constructor(globalConfig) {
```

Comment:
We should be defining the output file within the coverageDirectory defined in the jest configuration. Here are the options available via the global config: https://github.com/facebook/jest/blob/master/packages/jest-types/src/Config.ts#L243

```suggestion
  constructor(globalConfig) {
      super(globalConfig);
      const { coverageDirectory } = this.globalConfig;
      this.outputFile = path.resolve(process.cwd(), coverageDirectory, 'JestTestResults.txt');
```

Reply:
I'm wondering, do we want to scope this to terra dir? & do we want a unique naming pattern for these output files?

Reply:
to clarify, I'm thinking a naming conversation for this output.. @ryanthemanuel  and I talked about it yesterday about the output being `/tests/jest/reports/results/<filename>`

what are other's thoughts? Also do we want this to be a text file? Can we output different? this file will eventually be pulled into a site report.

Reply:
I know I had originally said to do a text file, but I think I like it being a json file to be more consistent with wdio.  I like this path:

/tests/jest/reports/results/result-output.json maybe?

Then the content of the json would be:

{
  "type": "jest",
  "output": <test output>
}

Then for the wdio we could do:

/tests/wdio/reports/results/result-<theme>-<locale>-<form-factor>.json

with contents:

{
  "type": "wdio",
  "theme": <theme>,
  "locale": <locaale>,
  "form-factor": <form-factor>,
  "output": <test-output>
} 

Any thoughts?

Reply:
ok